src/lib/autoinstall/autoinst_issues_presenter.rb

```diff
@@ -0,0 +1,130 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+Yast.import "HTML"
+Yast.import "RichText"
+
+module Y2Autoinstallation
```

Comment:
I would add a `FIXME` stating that we should merge this class with [StorageProposalIssuesPresenter](https://github.com/yast/yast-autoinstallation/blob/master/src/lib/autoinstall/storage_proposal_issues_presenter.rb) as there is a lot of duplicate code (I know that's your intention, I would just add a note).

Reply:
Yes, I will add it if you want. But how many FIXMEs do we already have in our code since years and no one takes care about it....?